server/server.go

```diff
@@ -455,19 +455,20 @@ func cutPluginCommandF(w http.ResponseWriter, slashCommand *MMSlashCommand, tag,
 		return nil
 	}
 
-	msg := fmt.Sprintf("Tag %s created. Waiting for the artifacts to sign and publish.\nWill report back when the process completes.\nGrab :coffee: and a :doughnut: ", tag)
+	cmdFlags := fmt.Sprintf("Flags: tag `%s`, repo `%s`, commitSHA `%s`, force `%v`.", tag, repo, commitSHA, force)
+	msg := fmt.Sprintf("%s\nTag created. Waiting for the artifacts to sign and publish.\nWill report back when the process completes.\nGrab :coffee: and a :doughnut: ", cmdFlags)
 	if err := createTag(ctx, client, Cfg.GithubOrg, repo, tag, commitSHA); errors.Is(err, ErrTagExists) {
 		if !force {
-			WriteErrorResponse(w, NewError(fmt.Errorf("Tag %s already exists, not generating any artifacts. Use --force to regenerate artifacts.", tag).Error(), nil))
+			WriteErrorResponse(w, NewError(fmt.Errorf("Tag already exists. Not generating any artifacts. Use --force to regenerate artifacts. ").Error(), nil))
 			return nil
 		}
-		msg = fmt.Sprintf("Tag %s exists. Waiting for the artifacts to sign and publish.\nWill report back when the process completes.\nGrab :coffee: and a :doughnut: ", tag)
+		msg = fmt.Sprintf("%s\nTag exists. Waiting for the artifacts to sign and publish.\nWill report back when the process completes.\nGrab :coffee: and a :doughnut: ", cmdFlags)
 	} else if err != nil {
 		WriteErrorResponse(w, NewError(err.Error(), nil))
 		return nil
 	}
 
-	WriteEnrichedResponse(w, "Plugin Release Process", msg, "#0060aa", IN_CHANNEL)
+	WriteEnrichedResponse(w, "Plugin Release Process, flags", msg, "#0060aa", IN_CHANNEL)
```

Comment:
It's ok to commit the flags part here, otherwise the heading will look odd.

This is how it currently looks

![image](https://user-images.githubusercontent.com/25732808/87972063-4d5afa80-ca94-11ea-802b-1ed49681d331.png)

```suggestion
	WriteEnrichedResponse(w, "Plugin Release Process", msg, "#0060aa", IN_CHANNEL)
```